composer.json

```diff
@@ -17,7 +17,7 @@
     },
     "require-dev": {
         "doctrine/data-fixtures": "^1.3",
-        "ergebnis/composer-normalize": "^1.2",
```

Comment:
The version required here actually installs `localheinz/composer-normalize`, see https://github.com/ergebnis/composer-normalize/blob/1.3.1/composer.json#L2.